subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -955,11 +961,6 @@ static int init_reset(void)
 	return 0;
 }
 
-static inline struct ll_adv_aux_set *aux_acquire(void)
-{
-	return mem_acquire(&adv_aux_free);
-}
-
```

Comment:
It seems inconsistent to drop `aux_acquire` but keep `aux_release`. Please either keep both (which I would prefer) or drop both.

Reply:
I will revert.